src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
```

Comment:
It seems like you're trying to style the `body` element, but you're using a class selector `.body`. If you want to style the `body` tag, you should use `body` without the dot. If you meant to style a class, make sure the class `body` is applied in your HTML.